osu.Game.Modes.Mania/ManiaRuleset.cs

```diff
@@ -102,6 +100,8 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
 
         public override FontAwesome Icon => FontAwesome.fa_osu_mania_o;
 
+        public override KeyCounter[] GameplayKeys => new KeyCounter[] { /* Todo: Should be keymod specific */ };

```

Comment:
hmm, should we rename KeyCounter to ButtonCounter or something? Key doesn't make too much sense when we have mouse buttons also being tracked...